checker/examples/units-extension/Expected.txt

```diff
@@ -0,0 +1,6 @@
+Demo.java:10: error: [assignment.type.incompatible] incompatible types in assignment.
+        frq = 5;
+              ^
+  found   : int
+  required: @Hz (@Hz :: int)
```

Comment:
The output in "required" is not what a human would expect.
Is this also the output for errors when used on the command line?
Shouldn't this be "@Hz int"?